src/handle-data.ts

```diff
@@ -23,19 +35,27 @@ export function getColumnOrder({ qColumnOrder, qDimensionInfo, qMeasureInfo }) {
  * @param {Number} numDims
  * @returns dimensions and measures total cell values as strings
  */
-export function getTotalInfo(isDim, layout, colIndex, numDims, columnOrder) {
+export function getTotalInfo(
+  isDim: boolean,
+  layout: TableLayout,
+  colIndex: number,
+  numDims: number,
+  columnOrder: number[]
+) {
```

Comment:
So the return type is now implied as `string | undefined` we probably want to enforce that it becomes a string

Reply:
I see that it is fine to have totalInfo in Column to be undefined

Reply:
hmm yeah I guess it is would work too. It needs to be added to the column type though, i think I saw it missing there

Reply:
yes, I added in my previous commit.